README.md

```diff
@@ -73,7 +68,7 @@ rsx! {
 You can easily declare your own custom widgets:
 ```rust
 #[widget]
-pub fn MyCustomWidget(children: Children) {
```

Comment:
@MrGVSV Can you make sure I'm not missing anything here? Feel free to commit changes directly to this branch.

Reply:
Yep, left a few suggestions. You can merge or discard them if you like.